jupyterlab_bigquery/src/components/details_panel/dataset_details_panel.tsx

```diff
@@ -24,17 +32,10 @@ export default class DatasetDetailsPanel extends React.Component<Props, State> {
       hasLoaded: false,
       isLoading: false,
       details: { details: {} } as DatasetDetails,
+      rows: [],
     };
   }
 
-  async componentDidMount() {
```

Comment:
We have a few of these empty `componentDidMount` functions scattered around, I've deleted this one and the one in `table_details_panel.tsx` since they aren't doing anything right now.